hoomd/md/pair/pair.py

```diff
@@ -1165,6 +1165,82 @@ def __init__(self, nlist, r_cut=None, r_on=0., mode='none'):
         self._add_typeparam(params)
 
 
+class SMie(Pair):
+    r"""Shifted Mie pair potential.
```

Comment:
TODO: Later I'll have some comments on how to update these docs to be consistent with recently merged and soon to be merged PRs that update this style.